packages/client/hmi-client/src/components/projects/ProjectCard.vue

```diff
@@ -1,44 +1,113 @@
 <script setup lang="ts">
-export interface Props {
-	name: string;
+import { Project, ProjectAssetTypes } from '@/types/Project';
+import Card from 'primevue/card';
+import Button from 'primevue/button';
+
+defineProps<{ project: Project }>();
+
+function formatTimeStamp(timestamp) {
+	const formattedDate = new Date(timestamp).toLocaleDateString(undefined, {
+		year: 'numeric',
+		month: 'short',
+		day: 'numeric'
+	});
+	return `Last updated ${formattedDate}`;
 }
```

Comment:
This can be put in the utils folder for dates, like a `lastUpdated(timestamp)`

Reply:
I'm not sure if we want to do that yet, as the date is formatted differently in different places

Reply:
It's okay, people can create different version in the utility